stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -83,16 +188,14 @@ extension Substring : _ObjectiveCBridgeable {
     return result != nil
   }
 
-  @_effects(readonly)
+  @_effects(releasenone)
```

Comment:
Maybe bring back these readonlys. They were removed while Karoy and I were debugging something weird.